mesheryctl/internal/cli/root/system/model.go

```diff
@@ -0,0 +1,364 @@
+// Copyright 2023 Layer5, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package system
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/layer5io/meshery/server/models"
+	"github.com/layer5io/meshkit/models/meshmodel/core/v1alpha1"
+	"github.com/manifoldco/promptui"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	// flag used to specify format of output of view {model-name} command
+	outFormatFlag string
+)
+
+// represents the `mesheryctl system model list` subcommand.
+var listModelCmd = &cobra.Command{
+	Use:   "list",
+	Short: "list models",
+	Long:  "list name of all registered models",
+	Example: `
+// View current provider
+mesheryctl system model list
+	`,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		//Check prerequisite
+		hcOptions := &HealthCheckOptions{
+			IsPreRunE:  true,
+			PrintLogs:  false,
+			Subcommand: cmd.Use,
+		}
+		hc, err := NewHealthChecker(hcOptions)
+		if err != nil {
+			return ErrHealthCheckFailed(err)
+		}
+		// execute healthchecks
+		err = hc.RunPreflightHealthChecks()
+		if err != nil {
+			cmd.SilenceUsage = true
+			return err
+		}
+		cfg, err := config.GetMesheryCtl(viper.GetViper())
```

Comment:
```suggestion
		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
```
Please Make sure you use similar naming